src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
   margin: 0;
+  display: flex;
+}
+
+.stopwatch {
+  --minute-duration: 60s;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  width: 80vmin;
+  height: 80vmin;
+  position: relative;
+
+  &--speed-up {
+    --minute-duration: 10s;
+  }
+
+  &__minutes-hand {
+    transform-origin: bottom;
+    animation: hand-rotate calc(var(--minute-duration) * 60) steps(60) infinite;
+    background-color: #0700ff;
+    width: 3vmin;
+    height: 20vmin;
+    position: absolute;
+    bottom: 40vmin;
+    left: 38.5vmin;
+    transform: rotate(0);
```

Comment:
The `transform-origin` property is duplicated in `.stopwatch__minutes-hand` and `.stopwatch__seconds-hand`. If both elements share the same `transform-origin`, consider creating a common class to avoid repetition and enhance maintainability.